src/test/java/com/testapp/subject/controller/SubjectControllerIntegrationTest.java

```diff
@@ -69,17 +76,17 @@ public void when_subject_created_it_should_show_in_the_list() throws Exception{
                         result -> MockMvcResultMatchers
                                 .xpath("//table//tbody//tr[0]//td[1]//a[contains(title(), 'math12')]")
                                 .exists());
-        List<Subject> subject = subjectRepository.findAll();
-        subjectRepository.deleteAll(subject);
+
     }
     @Test
     @WithMockUser(username = "test", password = "test")
     public void when_create_form_invalid_it_should_display_validation_errors() throws Exception{
```

Comment:
`when_title_is_empty_create_form_should_display_validation_error()`

Reply:
You also need to add one case where title is already exists, i.e.
`when_title_already_exists_create_form_should_display_validation_error()`, please do not forget to create subject with repository first and try to use same title on create form